NitroxServer/Server.cs

```diff
@@ -69,7 +70,25 @@ public void Save()
 
             IsSaving = true;
             NitroxConfig.Serialize(serverConfig); // This is overwriting the config file => server has to be closed before making changes to it
-            worldPersistence.Save(world, serverConfig.SaveName);
+            if (worldPersistence.Save(world, serverConfig.SaveName))
+            {
+                try
+                {
+                    string postSaveCommandPathTrimmed = serverConfig.PostSaveCommandPath.Trim('"');
+
+                    // Call external tool for backups, etc
+                    if (!string.IsNullOrEmpty(postSaveCommandPathTrimmed) ||
+                        File.Exists(postSaveCommandPathTrimmed))
+                    {
+                        Process.Start(postSaveCommandPathTrimmed);
```

Comment:
Recommend using the `FileSystem.OpenOrExecuteFile` function instead of process.start so that files are opened with default editor. Make sure to dispose of the Process object, as to not leak memory.

Reply:
FileSystem.OpenOrExecuteFile doesn't work with .bat on my system. I did dispose of the process now. Good catch.